src/App.css

```diff
@@ -1,38 +1,277 @@
+  font-size: 1rem;
+}
+
+/* Styles */
+
 .App {
   text-align: center;
+  background: url(/x-xtgONQzGgOE-unsplash.jpg) center;
+  background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+  height: 100vh;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.sweaterWeather{
+  font-size: 2rem;
+  color: yellow;
+  /* background: linear-gradient( yellow, orange );
+  -webkit-background-clip: text; */
+  /* -webkit-text-fill-color: transparent; */
+  font-family: 'Cabin', sans-serif; 
+  text-shadow: 1px 1px black;
+
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.notSweaterWeather{
+  font-size: 2rem;
+  color: black;
+  font-family: 'Cabin', sans-serif; 
+  text-shadow: 1px 1px yellow;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+/* Return Layout*/
+
+.tempIcon{
   display: flex;
-  flex-direction: column;
-  align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  padding: 0 2%;
+  align-items: center;
+}
+
+/* Input and Button */
+
+.submitButton{
+    margin: 0;
+    padding: 10px 30px;
+    border: none;
+    transition: 0.3s ease-in-out;
+    background: #FFF176;
+    color: black;
+}
+
+.submitButton:hover{
+    background: black;
+    color: #d64c31;
+}
+
+.submitButton:active{
+    background: black;
+    color: #d64c31;
+    color: white;   
+}
+
+.submitButton:focus{
+    outline: 5px solid black;     
+}
+  
+.cityInput{
+  text-align: center;
+  width: 50%;
+  padding: 10px 0;
+  border: none;
+}
+
+.cityInput:focus {
+  outline: 5px solid black;
+}
+
+label{ 
+  margin: 2% 0;
+  display:block; 
+}
+
+/* Footer */
+
+footer {
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color: black;
+    color: white;
+    width: 100%;
+    text-align: center;
+}
+
+.footerLinks {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+}
+
+footer .wrapper{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 auto;
+}
+
+a{
   color: white;
+  text-decoration: none;
 }
 
-.App-link {
-  color: #61dafb;
+a:hover,
+a:focus{
+  background: black;
+  color: #d64c31;
+}
+a:active{
+  color: #d64c31;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+/* Media Queries */
+
+@media (max-width: 950px) {
+  .searchAndButton{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 5%;
+  }
+  .cityInput{
+    width: 100%;
+    
   }
-  to {
-    transform: rotate(360deg);
+  .submitButton{
+    width: 100%;
   }
 }
+
+@media (max-width: 650px) {
+  .headerContainer{
+    flex-direction: column;
+  }
+  .sweaterWeather{
+    font-size: 1.5rem;
+  }
+}
+
+@media (max-width: 550px) {
+  .footerLinks{
+    margin-top: 5%;
+    flex-direction: column;
+  }
+}
+
+@media (max-width: 500px) {
```

Comment:
I think its great that you are using so many break point for media query. 
This should make resizing much more smooth I'd assume.